File: Modul 7/Soal 3/PRAK603-2210817220018-Piola Evania.c
#include <stdio.h>

int main()
{
    for(int i = 1; i < 4; i++){
        int b1, b2, isi;
        scanf("%d %d", &b1, &b2);
        int m1[b1], m2[b2];
        if(b1 == b2){
            for(int i = 0; i < b1; i++){
                scanf("%d", &isi);
                m1[i] = isi;
        }
        for(int i = 0; i < b2; i++){
            scanf("%d", &isi);
            m2[i] = isi;
        }
        for(int i = 0; i < b1; i++){
            int hasil = m1[i] * m2[i];
            printf("%d ", hasil);
        }
        }
        else{
            printf("Jumlah Tidak Sama");
        }
        printf("\n\n");
    }
}

File: Modul 7/Soal 3/PRAK603-2210817220018-Piola Evania.py
for i in range(3):
    b1, b2 = map(int, input().split())
    if b1 == b2:
        a1 = list(map(int, input().split()))
        a2 = list(map(int, input().split()))
        m1 = []
        m2 = []
        x = 0
        m1.append(a1[x : x + b1])
        m2.append(a2[x : x + b2])
        for i in range(0, b1):
            print(m1[0][i] * m2[0][i], end=' ')
    else:
        print("Jumlah Tidak Sama")
    print("\n\n")